scripts/cmake-configurator/parser/cmake_row.py

```diff
@@ -1,5 +1,13 @@
 import re
 from typing import Match, Optional
+from enum import Enum
+
+class types(Enum):
+   EMPTY: str = "EMPTY"
+   SLASH_COMMENT: str = "SLASH_COMMENT"
+   HASHTAG_COMMENT: str = "HASHTAG_COMMENT"
+   VARIABLE: str = "VARIABLE"
+   NOT_PARSABLE: str = "NOT_PARSABLE"
```

Comment:
Using string values for Enum items is **bad style**. See my comments below. Use numbers and keep in `type` an instance of the Enum class, not the string value.